tests/testcore/functional_test_base.go

```diff
@@ -193,7 +201,17 @@ func (s *FunctionalTestBase) SetupSuiteWithCluster(clusterConfigFile string, opt
 
 	// Logger might be already set by the test suite.
 	if s.Logger == nil {
-		s.Logger = log.NewTestLogger()
```

Comment:
I filed a ticket to port all `log.NewTestLogger()` calls to the new logger. Including xdc tests, too.

Reply:
assign it to me.

Reply:
iirc there was another test logger somewhere that did assertions.. should we replace that one too?

ah yeah: https://github.com/temporalio/temporal/blob/main/temporal/server_test.go#L188

Reply:
I'll add that to the ticket 👍